nvtabular/inference/triton/__init__.py

```diff
@@ -152,26 +174,73 @@ def convert_triton_output_to_df(columns, response):
     return cudf.DataFrame({col: response.as_numpy(col) for col in columns})
 
 
-def _generate_model_config(workflow, name, output_path):
+def _generate_model_config(
+    workflow, name, output_path, output_model=None, max_batch_size=None, cats=None, conts=None
+):
     """given a workflow generates the trton modelconfig proto object describing the inputs
     and outputs to that workflow"""
-    config = model_config.ModelConfig(name=name, backend="python")
 
-    for column, dtype in workflow.input_dtypes.items():
-        config.input.append(
-            model_config.ModelInput(name=column, data_type=_convert_dtype(dtype), dims=[-1, 1])
+    if max_batch_size is None:
+        config = model_config.ModelConfig(name=name, backend="python")
+    else:
+        config = model_config.ModelConfig(
+            name=name, backend="python", max_batch_size=max_batch_size
+        )
```

Comment:
I don't think we need the if statement here (like the default value for max_batch_size is None inside ModelConfig)
```suggestion
    config = model_config.ModelConfig(name=name, backend="python", max_batch_size=max_batch_size)
```